drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -380,6 +397,9 @@ static int sdw_compute_bus_params(struct sdw_bus *bus)
 		return -EINVAL;
 	}
 
+	mstr_prop->default_col = curr_dr_freq / mstr_prop->default_frame_rate /
+				 mstr_prop->default_row;
+
```

Comment:
Do we need to validate any of these are non 0 ? i.e. if they come from external ACPI or DISCO data ?

Reply:
> Do we need to validate any of these are non 0 ? i.e. if they come from external ACPI or DISCO data ?

In theory they should not be 0. SDW bus will not work if they are 0. But yeah, it is worthy to check it.